.gitignore

```diff
@@ -28,7 +28,7 @@ nbdist/
 **/.DS_Store
 
 upload/
-
+**/checkstyle.xml
```

Comment:
checkstyle is used to unify code standards, so it's not a local file, it shouldn't be ignored